python/pyarrow/parquet.py

```diff
@@ -2215,7 +2245,7 @@ def write_metadata(schema, where, metadata_collector=None, **kwargs):
         metadata.write_metadata_file(where)
 
 
-def read_metadata(where, memory_map=False):
+def read_metadata(where, memory_map=False, decryption_properties=None):
```

Comment:
I don't this it's useful to change this simple wrapper function. People can probably call `ParquetFile` directly.

Reply:
We thought of adding it for consistency